app/styles/_variables.scss

```diff
@@ -506,7 +507,7 @@ $overlay-background-color: rgba(0, 0, 0, 0.4);
   /** merge/rebase status indicators */
   --status-pending-color: #{$yellow-700};
   --status-error-color: #{$red-500};
-  --status-success-color: #{$green-500};
+  --status-success-color: #1a7f37;
```

Comment:
This color comes from Primer (see `--fgColor-success` on https://primer.style/foundations/primitives/color )